app/views/content/life-as-a-teacher/teaching-as-a-career/_categories.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="row">
+  <section class="category col-space-m">
+    <section class="container category__cards main-section">
+      <nav class="category__nav-cards">
+        <ul>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Change to a career in teaching",
+              description: "Find out how to bring your skills and experience from a previous career to life in the classroom.",
+              path: "/life-as-a-teacher/teaching-as-a-career/change-careers",
+              image: "static/images/content/hero-images/0029.jpg"
+            )) %>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Early career teachers (ECTs)",
+              description: "Find out about the support early career teachers get when they start teaching, including mentor support and training based on the early career framework (ECF).",
+              path: "/life-as-a-teacher/teaching-as-a-career/early-career-teachers"
+            )) %>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
```

Comment:
Presume we will look to move this to an inspiration component (will this be covered on a separate ticket?)?

Reply:
Good point, I'm not sure how we use the inspiration component so would rather we tackle that under a different ticket.